data/fin.lean

```diff
@@ -100,3 +100,6 @@ def lower_left {n m} (i : fin (n + m)) (h : i.val < n) : fin n :=
 
 def lower_right {n m} (i : fin (n + m)) (h: i.val >= n) : fin m :=
 ⟨i.val - n, by simp [nat.sub_lt_right_iff_lt_add h, i.is_lt]⟩
+
+def swap_sum {n m}  (i: fin (n + m)) : fin (m + n) :=
```

Comment:
I'm not sure about the name. Perhaps `swap_add` or `add_comm`? I don't think `sum` is generally used in mathlib to refer to `+`.

Reply:
I like swap_add.